packages/devtools/lib/src/ui/gtags.dart

```diff
@@ -0,0 +1,221 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
perhaps split the javascript wrapping code out into a separate library from the devtools specfic tracking / constants code?

Reply:
gtags.dart just the gtags wrapping and analytics.dart is the DevTools GA code.

Done.